user/serializers.py

```diff
@@ -1,24 +1,67 @@
-from django.contrib.auth import get_user_model
+from django.contrib.auth import get_user_model, authenticate
 from rest_framework import serializers
+from django.utils.translation import gettext as _
 
 
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = get_user_model()
         fields = ("id", "email", "password", "is_staff")
-        read_only_fields = ("is_staff",)
-        extra_kwargs = {"password": {"write_only": True, "min_length": 5}}
+        read_only_fields = ("id", "is_staff")
+        extra_kwargs = {
+            "password":
+                {
+                    "write_only": True,
+                    "min_length": 5,
+                    "style": {"inpute_type": "password"},
+                    "label": _("Password")
+                }
+        }
 
     def create(self, validated_data):
-        """Create a new user with encrypted password and return it"""
+        """create user with encrypted password"""
         return get_user_model().objects.create_user(**validated_data)
 
     def update(self, instance, validated_data):
-        """Update a user, set the password correctly and return it"""
         password = validated_data.pop("password", None)
         user = super().update(instance, validated_data)
+
         if password:
             user.set_password(password)
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.CharField(
```

Comment:
According to the checklist, you should not use 'serializers.CharField' for fields that are supposed to handle images. However, since this is an email field, it is correct to use 'serializers.CharField' here.